backend/playbackmanager.go

```diff
@@ -624,6 +624,14 @@ func (p *PlaybackManager) Volume() int {
 	return p.engine.CurrentPlayer().GetVolume()
 }
 
+func (p *PlaybackManager) SetSpeed(speed float64) {
```

Comment:
Would you be able to look into implementing the MPRIS support for this as well? There are Rate and SetRate stubs on the MPRIS handler here - https://github.com/dweymouth/supersonic/blob/main/backend/mpris.go#L242. Exporting Rate properly via MPRIS will keep the OS's concept of the playback position in sync with the actual position.

It would also be good to implement the same for Windows and Mac, though if you prefer, I can merge this PR to a feature branch once it's ready and then finish that myself. If nothing else, it can just give the OS an updated playback position once every 5 or 10 seconds to keep it from getting too out-of-sync.